docker/Dockerfile

```diff
@@ -94,12 +94,13 @@ RUN echo "hwloc_base_binding_policy = none" >> /usr/local/etc/openmpi-mca-params
     echo "rmaps_base_mapping_policy = slot" >> /usr/local/etc/openmpi-mca-params.conf && \
     echo "btl_tcp_if_exclude = lo,docker0" >> /usr/local/etc/openmpi-mca-params.conf
 
+WORKDIR /home/blueoil
```

Comment:
@kchygoe I changed `WORKDIR` directory here for `pip install`. It's OK/

Reply:
ah, I think it's okey but we have to check if it is okey to install setup.py without copying all other files.
If setup.py/cfg doesn't require `blueoil/*` files, we still need to install `blueoil` at the end of dockerfile

Reply:
Regarding `gast`,  let's clean up all requirements later ( I want to check all versions as well)

Reply:
@kchygoe I do not want to `pip install -e` several times.
How about to execute `COPY` before this line?

Reply:
I checked CI and `setup.py` works with the args. so can you just add `pip install -e .` after `COPY blueoil /home/blueoil/blueoil`? it would install blueoil itself